Updated LICENSE (#11483)
files-community/Files: NOTICE.md

# THIRD PARTY NOTICES

> Files uses following third-party libraries or other resources that may be distributed under licenses different than Files.

Reviewer: ```suggestion
Files uses third-party libraries and resources that may be distributed under licenses different than Files.
```

> In the event that we accidentally failed to list a required notice, please bring it to our attention. Post an issue on GitHub or send a message on Discord.

Reviewer: ```suggestion
In the event that we accidentally failed to list a required notice, please bring it to our attention by opening an issue.

```
